packages/ffe-core/less/typography.less

```diff
@@ -318,6 +318,16 @@
     margin: 0 4px;
 }
 
+.ffe-no-select {
+    user-select: none;
+    -webkit-touch-callout: none;
+}
+
+.ffe-select {
```

Comment:
Varfor trenger man denne klassen? Er det ikke samma sak som att ikke bruke `.ffe-no-select`?

Reply:
Ja, hvorfor ha to klasser som motvirker hverandre. Og hvorfor er en av dem en klasse som bare setter verdier til deres opprinnelige tilstand?

Noen komponenter kan være satt til å ikke være «selectable», da tenker jeg det kan være praktisk å ha en klasse som kan overkjøre dette dersom situasjonen krever det uten å måtte lage en spesifikk klasse for den komponenten.

Det er også mer oversiktlig å «toggle» mellom to klasser, istedet for å aktivere og deaktivere én. Hva skjer når klassen er deaktivert? Dette må utvikleren finne ut, og det er kanskje unødvendig tidsbruk. Er det to klasser ser man det fortere.

Mitt hovedpoeng er at det blir lettere for utviklerne å skjønne hva som skjer. Spesielt i et stort prosjekt der flere utviklere kommer og går. I et mindre prosjekt hadde jeg vektlagt annerledes.

Det er også ukjent for meg i hvilken grad elementer bør være «selectable» eller ikke. Er det flertallet som skal være «selectable» eller mindretallet(?). Dette spilte også en rolle når jeg la til to klasser, og ikke bare en.

Skal jeg lage et pull request med bare «.no-select» ?

Reply:
https://github.com/SpareBank1/designsystem/pull/785

> Edit: Closing this one for now. Going forward, let's discuss less obtrusive ways to fix this issue for web views.

Even om det nå er en klasse man kan bruke hvis man har lust, ønskar man att ting ser likt ut på tvers av team. Det kan hende det er riktigt att bruke denne klassen på mobil men det er då noe man må bli enig om. Kjeldigt svar og jag kanske ikke burde kommentert.